docs/amber_script.md

```diff
@@ -608,10 +609,14 @@ RUN {pipeline_name} \
 # data must be attached to the pipeline.
 
 # A start index of |value| will be used and the count of |count_value| items
-# will be processed.
+# will be processed. The draw is instanced if |inst_count_value| is greater
+# than one. In case of instanced draw |inst_value| controls the starting
+# instance ID.
 RUN {pipeline_name} DRAW_ARRAY AS {topology} \
     [ START_IDX _value_ (default 0) ] \
     [ COUNT _count_value_ (default vertex_buffer size - start_idx) ]
```

Comment:
I think this line needs a `\` on the end.

Reply:
done